packages/protocol/contracts/layer1/based/TaikoInbox.sol

```diff
@@ -43,37 +45,34 @@ abstract contract TaikoInbox is EssentialContract, ITaikoInbox, ITaiko {
         __Taiko_init(_owner, _rollupResolver, _genesisBlockHash);
```

Comment:
Client still needs `_state.slotA.genesisHeight` (https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/contracts/layer1/based/LibUtils.sol#L40) to know where to start scanning the L1 chain, seems its removed from the new `state` definition?